conjur/logic/login_logic.py

```diff
@@ -12,25 +12,28 @@
 
 # Internals
 from conjur.api.endpoints import ConjurEndpoint
-from conjur.errors import CertificateVerificationException
+from conjur.api.models import SslVerificationMetadata
+from conjur.errors import CertificateVerificationException, HttpSslError
 from conjur.interface.credentials_store_interface import CredentialsStoreInterface
 from conjur.wrapper.http_wrapper import invoke_endpoint, HttpVerb
 from conjur.data_object import CredentialsData, ConjurrcData
 
+
 class LoginLogic:
     """
     LoginLogic
 
     This class holds the business logic for populating the
     netrc configuration details needed to login to Conjur
     """
+
     def __init__(self, credentials_provider: CredentialsStoreInterface):
         self.credentials_provider = credentials_provider
 
     @classmethod
-    # pylint: disable=line-too-long,logging-fstring-interpolation
-    def get_api_key(cls, ssl_verify:bool, credential_data:CredentialsData,
-                    password:str, conjurrc:ConjurrcData) -> str :
+    # pylint: disable=logging-fstring-interpolation,try-except-raise,raise-missing-from
+    def get_api_key(cls, ssl_verification_metadata: SslVerificationMetadata, credential_data: CredentialsData,
+                    password: str, conjurrc: ConjurrcData) -> str:
```

Comment:
If line separation is needed between params, do it between each param:

```suggestion
    def get_api_key(cls,
                    ssl_verification_metadata: SslVerificationMetadata,
                    credential_data: CredentialsData,
                    password: str,
                    conjurrc: ConjurrcData) -> str:
```

Reply:
Done